MiniSim/MiniSim.swift

```diff
@@ -157,29 +162,50 @@ class MiniSim: NSObject {
                     UNUserNotificationCenter.showNotification(title: "Derived data has been cleared!", body: "Removed \(amountCleared) of data")
                     NotificationCenter.default.post(name: .commandDidSucceed, object: nil)
                 }
-            default:
-                break
             }
         }
     }
     
-    private func populateSections() {
-        if !menu.items.isEmpty {
-            return
+    private var devicesListHeaders: [NSMenuItem] {
```

Comment:
What do you think about moving this logic to `Menu.swift`? It seems to me like its a better place

Reply:
Sounds reasonable, but I will leave Clear Derived Data, Settings and Quit in the `MiniSim.swift` for now.